index-test.js

```diff
@@ -55,6 +55,82 @@ setTimeout(() => {
 
 
 
+setTimeout(() => {
+
+test("adding a new movie title to the list", () => {
+  
+  let movieTitle = document.querySelector(".movie-title")
+  //cheking the id attribute for the button
+  let bingeWorthy = document.querySelector(".movie-name");
+  //to fix the error of inputBtn not being defined
+  const inputBtn = document.querySelector('button')
+  //console.log(ThatGloriusMovie)
+  const creatingTheDiv = document.createElement('div')
+  creatingTheDiv.childElementCount;
+  //console.log(plusTitleToLife)
+   //picks up the input button on click
+   //const inputTitle  = document.querySelector(".list-input")
+  //picks up the input button on click
+  // defining the list input
+  const inputTitle = document.getElementsByClassName('list-input')
+   // giving the input a value otherwise new movie will be created
+  inputTitle.value = "Show me that new movie";
+  // what happens after the click.
+  const expected = inputTitle.childElementCount;
+  //checking the click of an item
+  // then user should then click the button.
+  //checking the click of an item
+  //adding a catcher at the beginning of the function seemed to fix the error of cannot read property of undefined.
+  if(this.item === undefined) {return}
+  let result = inputBtn[0].click();
+  equal(result, expected + 1);
+  const featureFilm = document.querySelector('div');
+  featureFilm.remove();
+  //time for resetting with new string.
+  inputTitle.value = ``;
+})
+
+}, 5000);
+
+setTimeout(() => {
+
+test("adding a second movie to the Binge list", () => {
+  //starting the new list from clicking the animated television
+  //const newList = document.querySelector('.main')
+  //to fix the error of inputBtn not being defined
+  const inputBtn = document.querySelector('button')
+  //InputDiv.childElementCount
+   //picks up the input button on click
+  const inputTitle  = document.querySelector(".list-input")
+  //const result = document.getElementsByClassName('input-list')
+  // then user should then click the button.
+  //checking the click of an item
+  //adding a catcher at the beginning of the function seemed to fix the error of cannot read property of undefined.
+  if(this.item === undefined) {return}
```

Comment:
This is interesting! I'm going to have a play around with it after the pull in my own console to understand it more :) Really enjoyed reading your code too!!